fabtests/common/shared.c

```diff
@@ -2913,7 +2913,7 @@ void eq_readerr(struct fid_eq *eq, const char *eq_str)
 
 int ft_sync()
 {
-	char buf;
+	char buf[8];
```

Comment:
This doesn't initialize buf.  Since only one byte is used, something like `char buf = 'a';` should work.